### PR TITLE
New package: keymapper-1.9.1

### LINKED ISSUE #36218: Package request: keymapper
A cross-platform context-aware key remapper.

image

- upstream url: https://github.com/houmain/keymapper
-  AUR url: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=keymapper

### PR DESCRIPTION
**Pull request by knusbaum**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

Fixes #36218

### CODE REVIEW

**Comment by knusbaum** [comment]
Need to fix cross-compiling.

File: srcpkgs/keymapper/files/keymapperd/run [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/bash
```
Comment (tibequadorian) [inline srcpkgs/keymapper/files/keymapperd/run:1]:
```suggestion
#!/usr/bin/sh
```

File: srcpkgs/keymapper/files/keymapperd/run [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/sh
```
Comment (tibequadorian) [inline srcpkgs/keymapper/files/keymapperd/run:1]: sorry i meant
```suggestion
#!/bin/sh
```

Comment (tibequadorian) [reply srcpkgs/keymapper/files/keymapperd/run:1]: That's what all other services have at least.

Comment (knusbaum) [reply srcpkgs/keymapper/files/keymapperd/run:1]: Right, I should've caught that.

**Comment by tibequadorian** [comment]
build logs say
```
CMake Warning at CMakeLists.txt:17 (message):
  Could not get version from git repository (defaulting to VERSION=0.0.0).
```
you should set it manually in `configure_args`

**Comment by tibequadorian** [comment]
Would be nice if you could get the musl patch upstreamed

**Comment by knusbaum** [comment]
@tibequadorian 
I'll fix the VERSION and put a patch up for musl for upstream.

**Pull request closed without merging**